plugins/exporter.koplugin/target/markdown.lua

```diff
@@ -0,0 +1,160 @@
+    self.settings = plugin_settings[self.name] or {}
+end
+
+local formatter_buttons = {
+    {_("None"), "none"},
+    {_("Bold"), "bold"},
+    {_("Bold Italic"), "bold_italic"},
+    {_("Italic"), "italic"},
+    {_("Strikethrough"), "strikethrough"},
+    {_("Underline (Markdownit style, with ++)"), "underline_markdownit"},
+    {_("Underline (with <u></u> tags)"), "underline_u_tag"},
+}
+
+function MarkdownExporter:editFormatStyle(drawer_style)
+    local radio_buttons = {}
+    for _idx, v in ipairs(formatter_buttons) do
+        table.insert(radio_buttons, {
+            {
+                text = v[1],
+                checked = self.settings.formatting_options[drawer_style] == v[2],
+                provider = v[2],
+            },
+        })
+    end
+    UIManager:show(require("ui/widget/radiobuttonwidget"):new{
+        title_text = _("Formatting style for ")  .. drawer_style,
+        width_factor = 0.8,
+        keep_shown_on_apply = true,
+        radio_buttons = radio_buttons,
+        callback = function(radio)
+            self.settings.formatting_options[drawer_style] = radio.provider
+        end,
+    })
+end
+
+function MarkdownExporter:populateSettings()
+    local changed = false
+    if self.settings.formatting_options == nil then
+        self.settings.formatting_options = {
+            lighten = "italic",
+            underscore = "underline_markdownit",
+            strikeout = "strikethrough",
+            invert = "bold",
+        }
+        changed = true
+    end
+    if self.settings.highlight_formatting == nil then
+        self.settings.highlight_formatting = true
+        changed = true
+    end
+    if changed then
+        self:saveSettings()
+    end
+end
+
+function MarkdownExporter:getMenuTable()
+    self:populateSettings()
+    return {
+        text = _("Markdown"),
+        checked_func = function() return self:isEnabled() end,
+        sub_item_table = {
+            {
+                text = _("Export to Markdown"),
+                checked_func = function() return self:isEnabled() end,
+                callback = function() self:toggleEnabled() end,
+            },
+            {
+                text = _("Format Highlights based on style"),
+                checked_func = function() return self.settings.highlight_formatting end,
+                callback = function() self.settings.highlight_formatting = not self.settings.highlight_formatting end,
+            },
+            {
+                text = _("Lighten"),
+                keep_menu_open = true,
+                callback = function()
+                    self:editFormatStyle("lighten")
+                end,
+            },
+            {
+                text = _("Strikeout"),
+                keep_menu_open = true,
+                callback = function()
+                    self:editFormatStyle("strikeout")
+                end,
+            },
+            {
+                text = _("Underscore"),
```

Comment:
In interface it is "Underline":
https://github.com/koreader/koreader/blob/4b0f7471f4a38c3371d01a8251d8f9f9f2937eb8/frontend/apps/reader/modules/readerhighlight.lua#L304

Reply:
FIxed. Thanks.